docroot/themes/custom/dcp/sass/components/_menu.scss

```diff
@@ -166,3 +166,79 @@ li a {
     margin-left: -45px;
   }
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
@pratik17893 

The way media query written in sass that's not the sass way.. media query should be written component wise.

e.g.

.region-bottom {
  @media (max-width: 768px) {
       padding-top: 10px;
   }
   .container {
      @media (max-width: 768px) {
           font-size: 20px;
       }
    }
}

And moreover, we must be using breakpoint variable for tablet so better we use the same to maintain the code consistency.